Infrastructure/Resources/Set-CDNComponents.ps1

```diff
@@ -0,0 +1,121 @@
+<#
+
+.SYNOPSIS
+Set additonal CDN (Content Delivery Network) Components
+
+.DESCRIPTION
+Set additional CDN (Content Delivery Network) Components
+
+.PARAMETER Source
+The source location of the files to be copied
+
+.PARAMETER Destination
+The blob destinaton of where to copy the files
+
+.PARAMETER SaSToken
+The SaS Token to access the blob storage container
+
+.PARAMETER StorageAccountName
+The StorageAccountName to apply the CORS settings
+
+.PARAMETER CDNProfileResourceGroup
+The Resource Group of the CDN
+
+.PARAMETER CDNProfileName
+The CDN Profile Name
+
+.PARAMETER CDNEndPointName
+The CDN EndPoint Name
+
+.PARAMETER PurgeContent
+The content to purge
+
+.EXAMPLE
+$DeploymentParameters = @ {
+    Source = "c:\FilesToBeCopied\"
+    Destination = "https://name.blob.core.windows.net/cdn"
+    SaSToken = "MySecureSaStokenString"
+    StorageAccountName = "mystorageaccountname"
+    CDNProfileResourceGroupName = "cdn"
+    CDNProfileName = "myprofile01"
+    CDNEndPointName = "myendpoint01"
+    PurgeContent = "/*",
+    EnableCORS = "Yes"
+}
+.\Set-CDNComponents.ps1 @DeploymentParameters
+
+#>
+# ---- Copy CDN content to blob storage and set MIME settings
+Param(
+    [Parameter(Mandatory = $true)]
+    [String]$Source,
+    [Parameter(Mandatory = $true)]
+    [String]$Destination,
+    [Parameter(Mandatory = $true)]
+    [String]$SaSToken,
+    [Parameter(Mandatory = $true)]
+    [String]$StorageAccountName,
```

Comment:
StorageAccountName and SaSToken should probably be in a parameter set otherwise if the user calls anything other than "Storage" for $OriginType they will still be forced to pass in these values.